src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -864,7 +864,7 @@ public CompletableFuture<Void> changeIssueStatus(ChangeIssueStatusParams params)
       client.showMessage(new MessageParams(MessageType.Info, "Issue status was changed"));
     }).exceptionally(t -> {
       lsLogOutput.errorWithStackTrace("Error changing issue status", t);
-      client.showMessage(new MessageParams(MessageType.Error, "Could not change status for the issue. Look at the SonarLint output for details."));
+      client.showMessage(new MessageParams(MessageType.Error, "Could not change status for the issue. Look at the SonarQube for IDE output for details."));
```

Comment:
I did not put `SonarQube for VS Code` here because the dropdown item in the output window will reflect name of the extension - `SonarQube for IDE`